cmd/m3db-operator/main.go

```diff
@@ -53,7 +57,7 @@ import (
 
 const (
 	// Informers will resync on this interval
-	_informerSyncDuration = time.Minute
+	_informerSyncDuration = 5 * time.Minute
```

Comment:
This is a fairly large increase, is it better to take a half measure to 2.5mins or something perhaps?

How often would we see a lost delivery do we think?

Reply:
I think it's pretty rare for us to see a lost delivery. We originally had this at a higher value but reduced it to 1min because we thought we were missing events, but it turns out we were overwhelming the controller.

For reference, the default value in kubebuilder is something like 6h and I've never seen issues with code using that.